libsddc/libsddc.cpp

```diff
@@ -326,42 +429,69 @@ int sddc_set_vhf_bias(sddc_t *t, int bias)
     return 0;
 }
 
-double sddc_get_sample_rate(sddc_t *t)
+uint64_t sddc_get_sample_rate(sddc_t *t)
 {
-    return 0;
+    switch(t->samplerateidx)
```

Comment:
sample rate is related to adc rate.

Reply:
Currently only two ADC rates supported in libsddc (64 and 128 MHz) and this relation handled here https://github.com/ik1xpv/ExtIO_sddc/blob/master/Core/RadioHandler.cpp#L154, isn't it?